Beam/app/Article.php

```diff
@@ -104,13 +108,17 @@ public function getVariantsCountAttribute(): array
             if (!$result->count) {
                 continue;
             }
-            $titleVariant = $result->tags->title_variant;
-            $imageVariant = $result->tags->image_variant;
-
-            if (!array_key_exists($titleVariant, $variants)) {
-                $variants[$titleVariant] = [];
+            if ($result->tags) {
```

Comment:
This condition is redundant, as tags are meant to be present there if `$result->count` has non-zero value.

Reply:
This was a mistake on my side. The tags don't need to be there - it was only our CMS tracking the variant titles even for non-AB-tested articles.